Makefile

```
@@ -2685,22 +2719,65 @@ endif
 INSTALL_LIBDIR_DYNLINK = $(INSTALL_LIBDIR)/dynlink
 
 # Installation
+
 .PHONY: install
-install:
+install::
 	$(MKDIR) "$(INSTALL_BINDIR)"
 	$(MKDIR) "$(INSTALL_LIBDIR)"
 	$(MKDIR) "$(INSTALL_STUBLIBDIR)"
 	$(MKDIR) "$(INSTALL_COMPLIBDIR)"
 	$(MKDIR) "$(INSTALL_DOCDIR)"
 	$(MKDIR) "$(INSTALL_INCDIR)"
 	$(MKDIR) "$(INSTALL_LIBDIR_PROFILING)"
-	$(INSTALL_PROG) $(runtime_PROGRAMS) "$(INSTALL_BINDIR)"
+
+ifeq "$(SUFFIXING)" "true"
+MANGLE_RUNTIME_NAME = $(TARGET)-$(1)-$(BYTECODE_RUNTIME_ID)$(EXE)
+MANGLE_RUNTIME_DLL_NAME = lib$(1)-$(TARGET)-$($(2)_RUNTIME_ID)$(EXT_DLL)
+else
+MANGLE_RUNTIME_NAME = $(1)$(EXE)
+MANGLE_RUNTIME_DLL_NAME = lib$(1)_shared$(EXT_DLL)
+endif
+
+define INSTALL_RUNTIME
+install::
+	$(INSTALL_PROG) \
+    runtime/$(1)$(EXE) \
+      "$(INSTALL_BINDIR)/$(call MANGLE_RUNTIME_NAME,$(1))"
+ifeq "$(SUFFIXING)" "true"
+	cd "$(INSTALL_BINDIR)" && \
+    $(LN) "$(TARGET)-$(1)-$(BYTECODE_RUNTIME_ID)$(EXE)" "$(1)$(EXE)"
+	cd "$(INSTALL_BINDIR)" && \
+    $(LN) "$(TARGET)-$(1)-$(BYTECODE_RUNTIME_ID)$(EXE)" \
+	    "$(1)-$(ZINC_RUNTIME_ID)$(EXE)"
```

A couple of lines are not starting with a Tab. I didn’t know `make` was accepting continuation lines not starting with Tab, but that breaks visual indentation for me. Did you mean to write:
```suggestion
	  runtime/$(1)$(EXE) \
	    "$(INSTALL_BINDIR)/$(call MANGLE_RUNTIME_NAME,$(1))"
ifeq "$(SUFFIXING)" "true"
	cd "$(INSTALL_BINDIR)" && \
	  $(LN) "$(TARGET)-$(1)-$(BYTECODE_RUNTIME_ID)$(EXE)" "$(1)$(EXE)"
	cd "$(INSTALL_BINDIR)" && \
	  $(LN) "$(TARGET)-$(1)-$(BYTECODE_RUNTIME_ID)$(EXE)" \
	    "$(1)-$(ZINC_RUNTIME_ID)$(EXE)"
```

---

That was intentional, indeed - in fact, I think check-typo might reject your suggestion? The tab is only required at the start of the recipe line itself. It should work if tabstop is set to 2 😉

---

Actually, looking again, it may have been intentional, but it's inconsistent! I've corrected them 🫣